cmake/darwin.cmake

```diff
@@ -56,7 +56,7 @@ IF ( NOT "${CMAKE_OSX_ARCHITECTURES}" MATCHES "ppc" AND NOT "${CMAKE_OSX_ARCHITE
     ADD_FLAGS(APPLE_CFLAGS -mtune=core2)
 ENDIF ( NOT "${CMAKE_OSX_ARCHITECTURES}" MATCHES "ppc" AND NOT "${CMAKE_OSX_ARCHITECTURES}" MATCHES "i386" )
 
-ADD_FLAGS(APPLE_CFLAGS -fexceptions -ffast-math -msse -fomit-frame-pointer)
```

Comment:
Please try `-msse2` instead of removing this if you have the time. If not: Have you found or filed a bug report in GCC? Could you link the bug-id/url?

Reply:
nice try, but does not work

existing  bug reports are here: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=78380